### PR TITLE
[DE] Update `Tournaments/OWC/2014`

### PR DESCRIPTION
**Pull request by respektive**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Tournaments/OWC/2014/de.md [hunk 1]
```diff
@@ -269,258 +263,249 @@ Dieses Turnier ist vorüber mit folgendem Ergebnis:
...
+3. Raumeinstellungen sind osu!, Team-Vs., Win Condition: 'Score'. Raumname muss "OWC 2014: TeamBlue vs TeamRed" lauten.
+4. Das Team, welches zuerst im Raumnamen genannt wurde, wird das blaue Team, das andere Team dementsprechend dann das rote Team.
+5. Den Spielern steht es frei 2 Beatmaps für die Aufwärmphase zu spielen. Beatmaps mit fragwürdigem Inhalt sind verboten.
+6. Jeder Kapitän kann 2 Beatmaps aus dem Pool wählen, die nicht gespielt werden sollen. Dies gilt für beide Teams und die komplette Partie hindurch.
+7. Beide Vetos müssen benutzt werden.
+8. Beide Kapitäne müssen `!roll` benutzen, um zu ermitteln, wer mit dem Bannen der beiden Beatmaps beginnen soll. Der andere Kapitän darf nicht dieselben Beatmaps bannen und muss daher andere Beatmaps auswählen, um Überlappungen zu vermeiden.
+9. Die Beatmaps werden abwechselnd von den beiden Kapitänen ausgewählt. Zuvor muss jedoch wieder mit `!roll` im `#multiplayer` ermittelt werden, welches Team zuerst anfängt.
+10. Die Kapitäne können frei nach Belieben eine Beatmap aus den einzelnen Gruppen auswählen.
+11. Im Falle eines Unentschiedens, muss die Tiebreaker Map gespielt werden.
```
Comment (The-Last-Cookie) [inline wiki/Tournaments/OWC/2014/de.md:483]:
```suggestion
11. Im Falle eines Unentschiedens, muss die Tiebreaker-Beatmap gespielt werden.
```

File: wiki/Tournaments/OWC/2014/de.md [hunk 2]
```diff
@@ -269,258 +263,249 @@ Dieses Turnier ist vorüber mit folgendem Ergebnis:
...
+12. Die Ergebnisse werden im osu!-Wiki und per Statistiken auf einer Webseite veröffentlicht.
 
 ### Mappoolanweisungen
 
 1. Es wird ein Mappool für die Gruppenphase, ein Mappool für das Achtelfinale, ein Mappool für das Viertelfinale, ein Mappool für Halbfinale und ein Mappool für das Finale geben.
 2. Jeder Mappool besteht aus 5 Gruppen: NoMod, HardRock, Hidden, DoubleTime und FreeMod
```
Comment (The-Last-Cookie) [inline wiki/Tournaments/OWC/2014/de.md:489]:
```suggestion
2. Jeder Mappool besteht aus 5 Stapeln: NoMod, HardRock, Hidden, DoubleTime und FreeMod
```
You've used this in the other article. Both is fine to me, jsut something that I wanted to mention.

File: wiki/Tournaments/OWC/2014/de.md [hunk 3]
```diff
@@ -269,258 +263,249 @@ Dieses Turnier ist vorüber mit folgendem Ergebnis:
...
 3. Jeder Mappool besteht aus 19 Maps.
```
Comment (The-Last-Cookie) [inline wiki/Tournaments/OWC/2014/de.md:490]:
```suggestion
3. Jeder Mappool besteht aus 19 Beatmaps.
```

File: wiki/Tournaments/OWC/2014/de.md [hunk 4]
```diff
@@ -269,258 +263,249 @@ Dieses Turnier ist vorüber mit folgendem Ergebnis:
...
-6. Die Hidden, HardRock und DoubleTime Gruppe wird mit dem entsprechenden, aktivierten Mod gespielt.
-7. Die FreeMod Gruppe schaltet den FreeMod frei. Jeder einzelne Spieler kann dann Hidden, HardRock, Flashlight oder mit gar keinen Mod spielen.
-   1. Spieler dürfen mehr als einen Mod auswählen.
-   2. Wenn eine FreeMod Map gespielt wird, müssen mindestens 2 Spieler jedes Teams mindestens einen Mod aktiviert haben.
-8. Die Tiebreaker Map wird mit denselben Bedingungen wie beim Freemod gespielt.
-   1. Wenn die Tiebreaker Map gespielt wird, muss jedoch niemand einen Mod aktiviert haben.
-9. Die Größe der NoMod Gruppe besteht aus 6 Maps in allen Etappen.
-10. Die Größe der mod-spezifischen Gruppen besteht aus 3 Maps in allen Etappen.
+4. Jeder Mappool hat eine Tiebreaker-Map.
+5. Die NoMod Gruppe wird ohne aktive Mods gespielt.
```
Comment (The-Last-Cookie) [inline wiki/Tournaments/OWC/2014/de.md:492]:
```suggestion
5. Der NoMod-Stapel wird ohne aktive Mods gespielt.
```

File: wiki/Tournaments/OWC/2014/de.md [hunk 5]
```diff
@@ -269,258 +263,249 @@ Dieses Turnier ist vorüber mit folgendem Ergebnis:
...
+6. Die Hidden, HardRock und DoubleTime Gruppe wird mit der entsprechenden, aktivierten Mod gespielt.
```
Comment (The-Last-Cookie) [inline wiki/Tournaments/OWC/2014/de.md:493]:
```suggestion
6. Der Hidden-, HardRock- und DoubleTime-Stapel wird mit der entsprechenden, aktivierten Mod gespielt.
```

File: wiki/Tournaments/OWC/2014/de.md [hunk 6]
```diff
@@ -269,258 +263,249 @@ Dieses Turnier ist vorüber mit folgendem Ergebnis:
...
+7. Die FreeMod Gruppe schaltet den FreeMod frei. Jeder einzelne Spieler kann dann Hidden, HardRock, Flashlight oder mit gar keiner Mod spielen.
```
Comment (The-Last-Cookie) [inline wiki/Tournaments/OWC/2014/de.md:494]:
```suggestion
7. Der FreeMod-Stapel schaltet den FreeMod frei. Jeder einzelne Spieler kann dann Hidden, HardRock, Flashlight oder mit gar keiner Mod spielen.
```

File: wiki/Tournaments/OWC/2014/de.md [hunk 7]
```diff
@@ -269,258 +263,249 @@ Dieses Turnier ist vorüber mit folgendem Ergebnis:
...
+8. Spieler dürfen mehr als eine Mod auswählen.
+9. Wenn eine FreeMod-Map gespielt wird, müssen mindestens 2 Spieler jedes Teams mindestens eine Mod aktiviert haben.
```
Comment (The-Last-Cookie) [inline wiki/Tournaments/OWC/2014/de.md:496]:
```suggestion
9. Wenn eine FreeMod-Beatmap gespielt wird, müssen mindestens 2 Spieler jedes Teams mindestens eine Mod aktiviert haben.
```

File: wiki/Tournaments/OWC/2014/de.md [hunk 8]
```diff
@@ -269,258 +263,249 @@ Dieses Turnier ist vorüber mit folgendem Ergebnis:
...
+10. Die Tiebreaker-Map wird mit denselben Bedingungen wie bei FreeMod gespielt.
```
Comment (The-Last-Cookie) [inline wiki/Tournaments/OWC/2014/de.md:497]:
```suggestion
10. Die Tiebreaker-Beatmap wird mit denselben Bedingungen wie bei FreeMod gespielt.
```

File: wiki/Tournaments/OWC/2014/de.md [hunk 9]
```diff
@@ -269,258 +263,249 @@ Dieses Turnier ist vorüber mit folgendem Ergebnis:
...
+11. Wenn die Tiebreaker-Map gespielt wird, muss jedoch niemand eine Mod aktiviert haben.
```
Comment (The-Last-Cookie) [inline wiki/Tournaments/OWC/2014/de.md:498]:
```suggestion
11. Wenn die Tiebreaker-Beatmap gespielt wird, muss jedoch niemand eine Mod aktiviert haben.
```

File: wiki/Tournaments/OWC/2014/de.md [hunk 10]
```diff
@@ -269,258 +263,249 @@ Dieses Turnier ist vorüber mit folgendem Ergebnis:
...
+12. Die Größe der NoMod Gruppe besteht aus 6 Maps in allen Etappen.
```
Comment (The-Last-Cookie) [inline wiki/Tournaments/OWC/2014/de.md:499]:
```suggestion
12. Die Größe des NoMod-Stapels besteht aus 6 Beatmaps in allen Etappen.
```

File: wiki/Tournaments/OWC/2014/de.md [hunk 11]
```diff
@@ -269,258 +263,249 @@ Dieses Turnier ist vorüber mit folgendem Ergebnis:
...
+13. Die Größe der mod-spezifischen Gruppen besteht aus 3 Maps in allen Etappen.
```
Comment (The-Last-Cookie) [inline wiki/Tournaments/OWC/2014/de.md:500]:
```suggestion
13. Die Größe der mod-spezifischen Stapel besteht aus 3 Beatmaps in allen Etappen.
```

File: wiki/Tournaments/OWC/2014/de.md [hunk 12]
```diff
@@ -269,258 +263,249 @@ Dieses Turnier ist vorüber mit folgendem Ergebnis:
...
 
 ### Zeitplananweisungen
 
 1. Jede Etappe wird **einzeln am Wochenende** abgehalten.
 2. Die Partien können sich in der Gruppenphase überlappen.
 3. Alle Double-Elimination Etappen finden entweder am Samstag oder am Sonntag statt.
```
Comment (The-Last-Cookie) [inline wiki/Tournaments/OWC/2014/de.md:506]:
```suggestion
3. Alle Doppeleleminierungsetappen finden entweder am Samstag oder am Sonntag statt.
```

File: wiki/Tournaments/OWC/2014/de.md [hunk 13]
```diff
@@ -269,258 +263,249 @@ Dieses Turnier ist vorüber mit folgendem Ergebnis:
...
-4. Die Zeiten der Partien werden vom Turnier Management festgelegt. Die Termine werden am Sonntag angekündigt, bevor die eigentliche Etappe stattfindet. Der Turnier Management bemüht sich die Zeiten so gut wie möglich an den Zeitzonen der teilnehmenden Spieler anzupassen.
-5. Die Kapitäne sind für die Anwesenheit deren Teams zuständig. Eine größere Teamgröße stellt sicher, dass jedes Team mindestens 4 Spieler in jeder Partie zur Verfügung hat. Wenn vom Team keine 4 Spieler anwesend sein sollten, zählt die Partie als aufgegeben.
+4. Die Zeiten der Partien werden vom Turnier-Management festgelegt. Die Termine werden am Sonntag angekündigt, bevor die eigentliche Etappe stattfindet. Das Turnier-Management bemüht sich die Zeiten so gut wie möglich an den Zeitzonen der teilnehmenden Spieler anzupassen.
```
Comment (The-Last-Cookie) [inline wiki/Tournaments/OWC/2014/de.md:507]:
```suggestion
4. Die Zeiten der Partien werden vom Turnier-Management festgelegt. Die Termine werden am Sonntag angekündigt, bevor die eigentliche Etappe stattfindet. Das Turnier-Management bemüht sich, die Zeiten so gut wie möglich an den Zeitzonen der teilnehmenden Spieler anzupassen.
```
I think there needs to be a comma. This is also the case for the other PR.

**Review by The-Last-Cookie — CHANGES_REQUESTED** [review]

File: wiki/Tournaments/OWC/2014/de.md [hunk 14]
```diff
@@ -269,258 +263,249 @@ Dieses Turnier ist vorüber mit folgendem Ergebnis:
...
 
 ## Regelsätze
 
 ### Turnierregeln
 
-1. Das osu! World Cup ist ein länderbezogenes 4 gegen 4 Team Turnier
-2. Die Beatmaps werden vom Map Wähler für jede Runde im Vorhinein am Sonntag angekündigt, bevor die eigentliche Partie stattfindet. Nur diese werden dann während den eigentlichen Partien zur Verfügung stehen.
-   - Eine Beatmap wird als Tiebreaker-Map mit angegeben. Diese Beatmap wird nur beim größtmöglichem Punktegleichstand gespielt.
+1. Der osu! World Cup ist ein länderbezogenes 4 gegen 4 Team Turnier
+2. Die Beatmaps werden vom Beatmapwähler für jede Runde im Vorhinein am Sonntag angekündigt, bevor die eigentliche Partie stattfindet. Nur diese werden dann während den eigentlichen Partien zur Verfügung stehen.
```
[the remark this answers is not in the document]
Comment (respektive) [reply wiki/Tournaments/OWC/2014/de.md:408]: Should probably be multiple actually.

File: wiki/Tournaments/OWC/2014/de.md [hunk 15]
```diff
@@ -269,258 +263,249 @@ Dieses Turnier ist vorüber mit folgendem Ergebnis:
...
-4. Um gültige und ernsthafte Registrierungen zu gewährleisten, wird jede Registrierung und Änderung vom Turnier Management geprüft.
-   1. Das Turnier Management kann Ihre Registrierung aufgrund von unzureichender Gameplay Fähigkeiten ablehnen.
-   2. Die Spieladminstration kann Ihre Registrierung aufgrund [vom Verstoß an den Nutzungsbedingungen](https://osu.zendesk.com/hc/en-us/articles/202090283-I-applied-to-play-in-an-official-tournament-but-was-denied-) ablehnen.
-5. Die maximale Anzahl an Teams liegt bei 32.
-6. Alle erfolgreichen Registrierungen der Teams wird nach der Registrierungsphase veröffentlicht.
-   - Kapitäne werden im Falle einer Akzeptierung oder Ablehnung benachrichtigt.
-7. Map Wähler dürfen nicht am Turnier als Spieler teilnehmen.
+   - Du bekommst eine Bestätigungsantwort, welche dir mitteilen soll, dass deine Anmeldung erfolgreich angekommen ist. Die Registrierung ist dann in Bearbeitung.
+7. Um gültige und ernsthafte Registrierungen zu gewährleisten, wird jede Registrierung und Änderung vom Turnier Management geprüft.
+8. Das Turnier Management kann Ihre Registrierung aufgrund von unzureichender Gameplay Fähigkeiten ablehnen.
```
[the remark this answers is not in the document]
Comment (respektive) [reply wiki/Tournaments/OWC/2014/de.md]: Personally think `Gameplay-Fähigkeiten` is better, even if it adds another hyphen :neutral_face:

File: wiki/Tournaments/OWC/2014/de.md [hunk 16]
```diff
@@ -269,258 +263,249 @@ Dieses Turnier ist vorüber mit folgendem Ergebnis:
...
-   | Viertelfinale | I, J, K, L & R, S, T, U |
-   | Halbfinale | M, N & V, W, X, Y, Z,AA |
-   | Finale - Woche 1 | O & AB, AC, AD, AE |
-   | Finale - Woche 2 | P, Q |
+4. Am meisten gewonnene Partien.
+5. Hat ein größeres `{(Anzahl an gewonnenen Maps) - (Anzahl an verlorenen Maps)}`.
+6. Am meisten gewonnene Maps.
+7. Hat ein größeres `∑{(gesamtes Ergebnisdifferenz) / (maximales Ergebnis)}`.
+8. Gewinner von Rückspielen.
+9. Die besten 2 Teams in der Gruppe kommen in die Double-Elimination Etappen.
```
[the remark this answers is not in the document]
Comment (respektive) [reply wiki/Tournaments/OWC/2014/de.md]: Hmmm, possible. Though the other article was mainly talking about knock-out rounds, but I think getting rid of the hyphens is always good xd

File: wiki/Tournaments/OWC/2014/de.md [hunk 17]
```diff
@@ -269,258 +263,249 @@ Dieses Turnier ist vorüber mit folgendem Ergebnis:
...
-2. Den Spielern steht es zu 2 Beatmaps für die "Warm-Up" Phase zu spielen. Beatmaps mit fragwürdigem Inhalt sind verboten.
-3. Jeder Kapitän kann 2 Beatmaps aus dem Pool wählen, die nicht gespielt werden sollen. Dies gilt für beide Teams und die komplette Partie hindurch.
-   1. Beide Vetos müssen benutzt werden.
-   2. Beide Kapitäne müssen `!roll` benutzen, um zu ermitteln, wer mit dem Verbannen der beiden Beatmaps beginnen soll. Der andere Kapitän darf nicht dieselben Beatmaps verbannen und muss daher andere Beatmaps auswählen, um Überlappungen zu vermeiden.
-4. Die Beatmaps werden abwechselnd von den beiden Kapitänen ausgewählt. Zuvor muss jedoch wieder mit `!roll` im `#multiplayer` ermittelt werden, welches Team zuerst anfängt.
-   1. Die Kapitäne können frei nach Belieben eine Beatmap aus den einzelnen Gruppen auswählen.
-   2. Im Falle eines Unentschiedens, muss die Tiebreaker Map gespielt werden.
-5. Die Ergebnisse werden im osu! Wiki und per Statistiken auf einer Webseite veröffentlicht.
+2. Der Raum wird verschlossen. Das Passwort und die Einladung zum Raum werden beiden Kaptiänen so früh wie möglich gesendet.
```
[the remark this answers is not in the document]
Comment (respektive) [reply wiki/Tournaments/OWC/2014/de.md]: ouch

File: wiki/Tournaments/OWC/2014/de.md (hunk 2 above)
Comment (respektive) [reply wiki/Tournaments/OWC/2014/de.md]: Was mainly just following the terms already used in this translation anyway. For specific mod pools there isn't really a proper term afaik, but lets go with this to stay consistent

File: wiki/Tournaments/OWC/2014/de.md [hunk 18]
```diff
@@ -1,127 +1,121 @@
...
-| ![Gold Crown](/wiki/shared/crown-gold.png "1st place") | ![][flag_JP] **Japan** |
-| ![Silver Crown](/wiki/shared/crown-silver.png "2nd place") | ![][flag_PL] **Polen** |
-| ![Bronze Crown](/wiki/shared/crown-bronze.png "3rd place") | ![][flag_DE] **Deutschland** |
+Der **osu! World Cup 2014** (***OWC 2014***) war ein länderbasiertes osu!-Turnier, das vom [osu!-Team](/wiki/People/The_Team) ausgerichtet wurde. Es war die fünfte Austragung des osu! World Cup.
 
 ## Turnierzeitplan
 
-| Event | Timestamp |
+| Event | Zeitpunkt |
 | :-- | :-- |
```
Comment (Walavouchey) [inline wiki/Tournaments/OWC/2014/de.md:16]:
```suggestion
| --: | :-- |
```

File: wiki/Tournaments/OWC/2014/de.md [hunk 19]
```diff
@@ -1,127 +1,121 @@
...
+| Viertelfinale | 2014-11-22 bis 2014-11-23 |
+| Halbfinale | 2014-11-29 bis 2014-11-30 |
+| Finalwoche 1 | 2014-12-06 |
+| Finalwoche 2 | 2014-12-13 |
 
 ## Preise
 
-| Placing | Prizes |
+| Platzierung | Preis(e) |
 | :-- | :-- |
```
Comment (Walavouchey) [inline wiki/Tournaments/OWC/2014/de.md:29]:
```suggestion
| :-: | :-- |
```

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**